Libraries/Image/Image.ios.js

```diff
@@ -88,6 +90,33 @@ var Image = React.createClass({
      * testing scripts.
      */
     testID: PropTypes.string,
+    /**
+     * onLoadingStart - Event callback function to invoke when image load begins.
```

Comment:
nit: can you remove `onLoadingStart -` from the beginning. We're displaying it twice on the docs and I sent an internal diff to remove the other ones from this file